R/empirical.R

```diff
@@ -1,25 +1,29 @@
-#' Obtain Empirical/Jackknife Covariance
+#' Obtain Empirical/Jackknife/BRL Covariance
```

Comment:
I think they are all kind of linearized method, but this BRL, Bias-Reduced Linearization method is kind of special because it use (-1/2) to reduce the bias. So calling it Linearized can be inappropriate. How about "Bias-Redued"?

Reply:
Ok Bias-Reduced works!